lib/openstack_query/runners/server_runner.py

```diff
@@ -23,11 +23,34 @@ class ServerRunner(QueryRunner):
     ServerRunner encapsulates running any openstacksdk Server commands
     """
 
-    def _run_query(
+    def _parse_meta_params(
         self,
         conn: OpenstackConnection,
-        filter_kwargs: Optional[Dict[str, str]] = None,
         from_projects: Optional[List[ProjectIdentifier]] = None,
+    ):
+        """
+        This method is a helper function that will parse a set of query meta params related to openstack server queries.
+        :param conn: An OpenstackConnection object - used to connect to openstack and parse meta params
+        """
+        try:
+            return {
```

Comment:
IMO this try is too large, can we just do

```
try:
   project = conn.identity.find_project(...)
except:
   ...

return { ... }
```

To really restrict the scope to the only step that should fail